requirements.txt

```diff
@@ -0,0 +1,10 @@
+prettytable
+requests
+yattag
+pyserial>2.5
+jsonmerge
+jsonschema
```

Comment:
Add restriction to jsonschema version number used by jsonmerge as well (I think that only supports a certain version).

Reply:
let's do it when we know which version is required. I didn't recognise any specific version that is required by icetea.

Reply:
If `jsonmerge` and `jsonschema` have internal dependencies, they should list those.
It is not our job to find out.. (Or should not be)

So, I think it is fine like this, if Icetea don't expect particular version of either tool.